.github/workflows/silences-report-expired.yaml

```
@@ -0,0 +1,13 @@
+name: Report expired silences
+
+on: [push, pull_request]
+
+jobs:
+  silences_report_expired:
+    runs-on: ubuntu-latest
+    name: Report expired silences
+    steps:
+      - uses: actions/checkout@v4
+      - uses: giantswarm/management-cluster-bases/.github/actions/silences-report-expired@silences-ci
```

Reminder: revert to main

```suggestion
      - uses: giantswarm/management-cluster-bases/.github/actions/silences-report-expired@main
```